test/matrices.jl

```diff
@@ -0,0 +1,10 @@
+@testset "Matrices" begin
+  # tests with Aitchson's definitions
+  d = 10
+  D = d+1
+  @test JColumn(d) == ones(d)
+  @test JMatrix(d) == ones(d, d)
+  @test F(d) == [I(d) -ones(d)]
+  @test maximum(abs.(G(D) - I(D)  + JMatrix(D)/ D)) < 1e-5
```

Comment:
What about `norm(G(D) - I(D) + J(D) / D, Inf) < 1e-5`? The function `norm` is available in LinearAlgebra.